kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabDhcpNatUi.java

```diff
@@ -213,6 +213,11 @@ public void getUpdatedNetInterface(GwtNetInterfaceConfig updatedNetIf) {
         }
     }
 
+    @Override
```

Comment:
Please change the header to: " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates"